packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -346,7 +346,11 @@ export interface Volume {
    * A name for the volume
    */
   name?: string;
-  // FIXME add dockerVolumeConfiguration
+
+  /**
+   * Specifies this configuration when using Docker volumes
+   */
+  dockerVolumeConfiguration?: DockerVolumeConfiguration;
```

Comment:
I wonder if it makes sense to have these inline in the `Volume` struct.

As well as, why are the name and the configuration optional? Does it make sense to leave them optional?

I know @SoManyHs was working on a higher-level abstraction over volumes, so I'd like her to weigh in as well.

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-volumes.html
Thanks, I changed `name` to required.